bridge/cli/bridge.py

```diff
@@ -19,25 +27,24 @@ def main():
         help="Platform where you want to deploy this app",
         choices=["render", "railway", "heroku"],
     )
-    init_parser.add_argument(
-        "--wsgi-path",
-        help="Path to your WSGI application callable (ex: myapp.wsgi:application)",
-        required=False,
-    )
-    init_parser.add_argument(
-        "--asgi-path",
-        help="Path to your ASGI application callable (ex: myapp.asgi:application)",
-        required=False,
-    )
+
+    # Parser for db
+    subparsers.add_parser("db", help="Open a database shell")
+
+    # Parser for redis
+    subparsers.add_parser("redis", help="Open a redis shell")
 
     # Parse the arguments
     args = parser.parse_args()
+    framework = detect_framework()
 
+    # TODO: pattern for additional config from the CLI
     if args.command == "init":
-        # Additional validation for the arguments
-        if args.command == "init" and args.wsgi_path and args.asgi_path:
-            parser.error("Both WSGI and ASGI paths cannot be provided")
-        initialize_platform(args.init_platform)
+        initialize(framework=framework, platform=args.init_platform)
```

Comment:
Might be good to namespace these as `db shell` to leave room for `db logs`, `db dump`, etc.